resources/notes.py

```diff
@@ -0,0 +1,18 @@
+from flask_restful import Resource, reqparse
+from models.notes import NotesModel
+from utils.authorizations import pm_level_required
+from schemas.notes import NotesSchema
+
+
+class Notes(Resource):
+    parser = reqparse.RequestParser()
+    parser.add_argument("text")
+    parser.add_argument("authorID")
+
+    @pm_level_required
+    def post(self, id):
+        data = Notes.parser.parse_args()
+        note_payload = {"ticketid": id, "text": data.text, "userid": data.authorID}
+
+        created_note = NotesModel.create(schema=NotesSchema, payload=note_payload)
+        return created_note.json()
```

Comment:
I think this can be changed to:
```python
class Notes(Resource):
    @pm_level_required
    def post(self, id):
        return NotesModel.create(schema=NotesSchema, payload=request.json).json()
```

The frontend code will need to make sure they're sending the right request.

POST("userid": ?, "ticketid": ?, "text": "some text")

As the backend is not responsible for translating request parameters.

Although I'd argue that the frontend shouldn't be sending the "userid", because the backend knows who the current user is and can assign appropriately. Let me know if you'd like to talk more about this.

Reply:
I love the idea of pulling the userid directly from the jwt, but I haven't been able to find out how to pull it from the request.  Do you know, or know where I can look to find that information?

Is it redundant to send the ticketid twice?  Inside the POST request as well as in the endpoint?  Would this make sense?

`class Notes(Resource):
    @pm_level_required
    def post(self, ticketid):
        return NotesModel.create(schema=NotesSchema, payload={**request.json, ticketid: ticketid}).json()`

Reply:
@NickSchimek  No rush, but I was curious if you saw my response to your comment.  I'm excited to get this endpoint finished so I can finish this issue and work on the rest of the Note functionality.  Thanks!

Reply:
Sorry for the delayed response. Your code:
```python
class Notes(Resource):
    @pm_level_required
    def post(self, ticketid):
        return NotesModel.create(schema=NotesSchema, payload={**request.json, ticketid: ticketid}).json()
```

Looks correct. good catch. The payload would need to include the `ticket_id`. It is not redundant since I don't think `request.json` should include the `ticket_id`.

It looks like we can get the current user by calling `get_jwt_identity()` - you may want to look up that method to confirm.
https://github.com/codeforpdx/dwellinglybackend/blob/acc9a65fbdb9b9d3b43250170f6f30b250f3db57/resources/user.py#L229

Reply:
```python
get_jwt-identity()
python```

Was exactly what I was looking for.  Thank you for that.  I believe I made all of the necessary changes.  Please let me know if there are any other improvements I can make.  Thanks!